test/NeinLinq.Fakes/DynamicQuery/Dummy.cs

```diff
@@ -11,5 +11,7 @@ public class Dummy
         public decimal? Number { get; set; }
 
         public Guid Reference { get; set; }
+
+        public DummyEnum Enum { get; set; }
```

Comment:
Not sure, if it matters, but I'd add additional testing for `public DummyEnum? OptionalEnum { get; set; }` (or the like) too